builder.Dockerfile

```diff
@@ -1,15 +1,16 @@
+ARG OWNER=alphagov
 ARG RUBY_MAJOR
-FROM ghcr.io/alphagov/govuk-ruby-base:${RUBY_MAJOR}
+FROM --platform=$TARGETPLATFORM ghcr.io/${OWNER}/govuk-ruby-base:latest
```

Comment:
Using an `${OWNER}` variable here so I can build these images under my account as well.

Reply:
Ugh, also left `latest` in here...